src/devices/machine/am9513.cpp

```diff
@@ -1251,9 +1255,15 @@ void am9513_device::command_write(u8 data)
 			if (BIT(data, c))
 			{
 				if (!BIT(data, 5))
+				{
+					LOGMASKED(LOG_MODE, "Disarm Counter %d\n", c + 1);
 					disarm_counter(c);
+				}
 				if (!BIT(data, 6))
+				{
+					LOGMASKED(LOG_MODE, "Save Counter %d\n", c + 1);
 					save_counter(c);
+				}
```

Comment:
`LOG_MODE` is for logging the master mode control setup.  It’s inappropriate to use here.

Also, there are already `LOG_GENERAL` messages in both `disarm_counter` and `save_counter` so these are unnecessary.